oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingMutationService.java

```diff
@@ -113,18 +121,27 @@ private EBPFProfilingTaskCreationResult buildError(String msg) {
     }
 
     private String checkCreateRequest(EBPFProfilingTaskFixedTimeCreationRequest request) throws IOException {
-        String err = "";
+        String err = null;
 
-        // validate process finder
-        if (request.getProcessFinder() == null) {
-            return "The process finder could not be null";
-        }
-        if (request.getProcessFinder().getFinderType() == null) {
-            return "The process find type could not be null";
+        err = requiredNotEmpty(err, "service", request.getServiceId());
+
+        // the service must have processes
+        if (err == null && getMetadataQueryDAO().getProcessesCount(request.getServiceId(), null, null) <= 0) {
+            err = "current service haven't any process";
         }
-        switch (request.getProcessFinder().getFinderType()) {
-            case PROCESS_ID:
-                err = requiredNotEmpty(err, "process id", request.getProcessFinder().getProcessId());
+
+        // the process label must be existed
+        if (err == null && CollectionUtils.isNotEmpty(request.getProcessLabels())) {
+            final List<String> existingLabels = getServiceLabelDAO().queryAllLabels(request.getServiceId());
+            List<String> notExistLabels = new ArrayList<>(existingLabels.size());
+            for (String processLabel : request.getProcessLabels()) {
+                if (!existingLabels.contains(processLabel)) {
+                    notExistLabels.add(processLabel);
+                }
+            }
+            if (notExistLabels.size() > 0) {
+                err = String.format("The label %s are not exist", Joiner.on(", ").join(notExistLabels));
```

Comment:
```suggestion
                err = String.format("The service doesn't have processes with label(s) %s.", Joiner.on(", ").join(notExistLabels));
```

Trying to make the statement more clear.